step.yml

```diff
@@ -1,7 +1,44 @@
 title: Manage iOS Code Signing
-summary: TODO
+summary: Automatically manages your code signing files.
 description: |-
-  TODO
+  The **Manage iOS Code Signing** Step takes care of automatically code signing your project before it's built on Bitrise.
+  The Step:
+  - generates, updates and downloads the provisioning profiles needed for your iOS project.
+  - verifies if your project is registered with the App Store Connect. If it was not, the Step registers your project.
+  - registers the iOS devices connected to your Bitrise account with the App Store Connect.
+  Use the **Manage iOS Code Signing** Step instead of the `xcodebuild` powered **Xcode Archive & Export for iOS** Step to manage your code signing assets if, for example:
+  - You use fastlane for your project.
+  - You use the **Ionic Archive** or the **Cordova Archive** build Steps in your project.
+  - You use a **Script** Step because your project has its own build scripts.
+  The **Manage iOS Code Signing** Step takes care of automatically code signing your project before it's built on Bitrise.
```

Comment:
Duplicated sentences here, is it intentional?

Reply:
no, it wasn't :(